requirements.txt

```
@@ -1 +1 @@
-numpy >= 1.13.3
+numpy >= 1.19.3 # keep in sync with NPY_TARGET_VERSION (setup.py)
```

Similarly the version constraint here could be dropped if we just want to support the oldest NumPy available on Python 3.9. Unless of course there are other reasons than NumPy C API to set a NumPy minimum

```suggestion
numpy
```